dnsimple/services.go

```diff
@@ -0,0 +1,93 @@
+package dnsimple
+
+import (
+	"fmt"
+)
+
+// ServicesService handles communication with the service related
+// methods of the DNSimple API.
+//
+// See https://developer.dnsimple.com/v2/services/
+type ServicesService struct {
+	client *Client
+}
+
+// ServiceSetting represents a single group of settings for a DNSimple Service.
+type ServiceSetting struct {
+	Name        string `json:"id,omitempty"`
```

Comment:
Replace `id` with `name`. That's why `name` was empty string and not `"username"` 😉